dashboard/config/routes.rb

```diff
@@ -388,6 +388,60 @@
     # these routes use course_course_name to match generated routes below that are nested within courses
     get '/courses/:course_course_name/guides/edit', to: 'reference_guides#edit_all', as: :edit_all_reference_guides
 
+    # Repeated routes to support Unit routes in both /s/ and /courses/xxx/unit/y
+    unit_routes = lambda do
```

Comment:
`unit_routes` groups up all the routes previously served by `/s/`. By grouping it like this, we can guarente we have the same routes for both `/s/...` and `/courses/.../units/...`